docs/pages/branding/careers.tsx

```diff
@@ -0,0 +1,551 @@
+          }}
+        />
+        <Typography
+          component="h3"
+          variant="h5"
+          color="primary"
+          fontWeight="extraBold"
+          sx={{ mb: 2 }}
+        >
+          Engineering
+        </Typography>
+        <Role
+          title="Full-stack Engineer"
+          description="You will initiate the development of a bold new product vertical. We are looking for an experienced and ambitious full-stack engineer that is ready to work in an entrepreneurial environment. You are a manager of one, you are curious, enjoy taking risks, and learning."
+          url="https://material-ui.com/company/full-stack-engineer/"
+        />
+        <Divider
+          sx={{
+            my: { xs: 1, sm: 2 },
+            borderColor: (theme) =>
+              theme.palette.mode === 'dark' ? 'primaryDark.600' : 'grey.100',
+          }}
+        />
+        <Role
+          title="React Engineer"
+          description="You will support the advanced components team, build new ambitious complex features, work on strategic problems, and help grow the adoption of the free open-source tier (freemium/open-core business model)."
+          url="https://material-ui.com/company/react-engineer/"
+        />
+        <Divider
+          sx={{
+            my: { xs: 1, sm: 2 },
+            borderColor: (theme) =>
+              theme.palette.mode === 'dark' ? 'primaryDark.600' : 'grey.100',
+          }}
+        />
+        <Typography
+          component="h3"
+          variant="h5"
+          color="primary"
+          fontWeight="extraBold"
+          sx={{ mb: 2 }}
+        >
+          Product
+        </Typography>
+        <Role
+          title="Technical Product Manager"
+          description="You will define and maintain the product roadmap for the advanced components, identify opportunities, define specs, and work with engineers to execute on the features. Experience as an engineer is essential for this role, as you will also contribute to development work in the beginning."
+          url="https://material-ui.com/company/technical-product-manager/"
+        />
+        <Divider
+          sx={{
+            my: { xs: 1, sm: 2 },
+            borderColor: (theme) =>
+              theme.palette.mode === 'dark' ? 'primaryDark.600' : 'grey.100',
+          }}
+        />
+        <Role
+          title="Product Manager"
+          description="You will initiate the exploration of a bold new product vertical. We are looking for an experienced and ambitious product manager who is ready to work in an entrepreneurial environment. You are a manager of one, you are curious, enjoy taking risks, and learning."
+          url="https://material-ui.com/company/product-manager/"
+        />
+        <Divider
+          sx={{
+            my: { xs: 1, sm: 2 },
+            borderColor: (theme) =>
+              theme.palette.mode === 'dark' ? 'primaryDark.600' : 'grey.100',
+          }}
+        />
+        <Role
```

Comment:
Would be great if we can extract the roles in an array and just display them here. It would be easier to change them in the future...

Reply:
I agree. Done!